### ISSUE TITLE
Incorrectly calculates "negative" css variable

### ISSUE THREAD

**Issue by anthonysapp** [ben-rogerson/twin.macro#410]
**1. The buggy behavior you’re experiencing and what should be happening instead.**
I define a css variable in my GlobalStyles like so :
```js
const CustomStyles = createGlobalStyle`
  :root{
    --gutter:12px;
    --gutter-half:6px;
  }
  @media (min-width: ${theme`screens.md`}) {
    :root{
      --gutter:16px;
      --gutter-half:8px;
    }
  }
  @media (min-width: ${theme`screens.lg`}) {
    :root{
      --gutter:20px;
      --gutter-half:10px;
    }
  }
`;

const GlobalStyles = () => (
  <>
    <BaseStyles />
    <CustomStyles />
  </>
);
```

Define grid row / column classes like so:
```js
const Container = tw.div`container mx-auto px-gutter-1/2`;
const Grid = {
  Row: ({ nowrap, ...rest }) => (
    <div
      css={[
        tw`relative flex -mx-gutter-1/2`,
        nowrap ? tw`flex-nowrap` : tw`flex-wrap`,
      ]}
      {...rest}
    />
  ),
  Column: ({ flex, ...rest }) => (
    <div
      css={[tw`relative p-gutter-1/2`, flex ? tw`flex` : tw`block`]}
      {...rest}
    />
  ),
};
```
... 
```js
<Container>
  <h1>Title</h1>
  <Grid.Row>
    <Grid.Column>Some info in a column</Grid.Column>
  </Grid.Row>
</Container>
```

Resulting css margins for the Grid.Row component (negative symbol in front of `calc`)
```css
margin-left: -calc(var(--gutter-half) * -1);
margin-right: -calc(var(--gutter-half) * -1);
```

**2. If you're using styled-components, emotion, goober, stitches or something else.**
Styled components / Next.js

**3. A link to a repo or a minimal demo that shows the bug in action.**
https://github.com/anthonysapp/twin-example

**Comment by ben-rogerson** [comment]
Thanks for the great bug report.

I've found and fixed the issue in #411

**Comment by anthonysapp** [comment]
Thanks for the great work, can't wait to try it out in production.
